packages/jest-cli/src/openCoverageReport.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @flow
+ */
+'use strict';
+
+const opn = require('opn');
+const {GOOGLE_CHROME} = require('./constants');
+
+const openFileInBrowser = (url: string) => {
+  try {
+    const options = {app: GOOGLE_CHROME};
+    opn(url, options).catch(() => {}); // Prevent `unhandledRejection` error.
```

Comment:
can you do `await opn(url, options);` and make this function async? This way we won't need the catch call.